test/com/ableton/VirtualEnvTest.groovy

```diff
@@ -177,7 +168,9 @@ class VirtualEnvTest extends BasePipelineTest {
     helper.addShMock(mockScriptCall, 'mock output', 0)
     helper.registerAllowedMethod('isUnix', []) { return true }
 
-    new VirtualEnv(script, 'python').run('mock-script')
+    VirtualEnv venv = new VirtualEnv(script)
+    venv.randomSeed = 1
+    venv.run('mock-script')
```

Comment:
Why is this split over 3 lines?

Reply:
I think this was a remnant of a previous implementation of the random seed. Same goes for the field. I've fixed both things now.